packages/inter-protocol/test/psm/test-psm.js

```diff
@@ -338,6 +338,27 @@ test('limit', async t => {
   // t.throwsAsync(() => await E(seat1).getOfferResult());
 });
 
+test('limit is for minted', async t => {
+  const { minted, anchor } = t.context;
+  // only 50 minted allowed per anchor
+  const anchorPerMinted = makeRatio(50n, anchor.brand, 100n, minted.brand);
+  const driver = await makePsmDriver(t, { anchorPerMinted });
+
+  trace('test going over limit');
+  const giveTooMuch = anchor.make(MINT_LIMIT);
+  const seat1 = await driver.swapAnchorForMintedSeat(giveTooMuch);
+  t.throwsAsync(() => E(seat1).getOfferResult(), {
+    message: 'Request would exceed mint limit',
+  });
+  trace('limit is enforced on the Minted rather than Anchor');
```

Comment:
use ava's message arg instead of trace?

```suggestion
  t.throwsAsync(() => E(seat1).getOfferResult(), {
    message: 'Request would exceed mint limit',
  }, 'limit is enforced on the Minted rather than Anchor');
```

Reply:
+1

Reply:
`trace` here is wrapped around AVA's log (in tests). This provided consistent reporting along with any `trace` statements in the stuff being tested (which are using `console`)